CHANGELOG.md

```diff
@@ -2,18 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
-## [7.1.0] 2022-07-01
-
-[Release link on GitHub](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0)
+## [7.1.0](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0) 2022-07-01
```

Comment:
```
## [7.1.0](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0) 2022-07-01
```
gives the following:
> ## [7.1.0](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0) 2022-07-01

I'd prefer the following:
```suggestion
## [[7.1.0](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0)] 2022-07-01
```
> ## [[7.1.0]](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0) 2022-07-01

Reply:
done in c02e38c